examples/platform/silabs/BaseApplication.cpp

```diff
@@ -188,6 +188,17 @@ void BaseApplicationDelegate::OnCommissioningWindowClosed()
         }
     }
 #endif // CHIP_CONFIG_ENABLE_ICD_SERVER && SLI_SI917
+    if (BaseApplication::GetProvisionStatus())
+    {
+#ifdef DISPLAY_ENABLED
+#ifdef QR_CODE_ENABLED
+        SilabsLCD::Screen_e screen;
+        slLCD.GetScreen(screen);
+        VerifyOrReturn(screen == SilabsLCD::Screen_e::QRCodeScreen);
```

Comment:
What happens if the screen is showing the "information" screen? Is the decision to leave it as is?

Reply:
Yes, this is only for the transition from the QR code screen to Demo UI screen. If it is information screen then it remains there.